include/vcpkg/sourceparagraph.h

```diff
@@ -214,8 +214,8 @@ namespace vcpkg
         std::string registry_location;
     };
 
-    void print_error_message(Span<const std::unique_ptr<ParseControlErrorInfo>> error_info_list);
-    inline void print_error_message(const std::unique_ptr<ParseControlErrorInfo>& error_info_list)
+    void print_error_message(View<LocalizedString> error_info_list);
```

Comment:
I did this because I saw callers who were wrapping perfectly good LocalizedString before calling this function which promptly unwrapped the perfectly good LocalizedString. But I can add the other overloads back.